modules/geolocation/geolocation.py

```diff
@@ -156,25 +176,35 @@ def __get_perspective_transform_matrix(
                 dst,
             )
         # All exceptions must be caught and logged as early as possible
-        # pylint: disable-next=bare-except
-        except:
-            # TODO: Logging
+        # pylint: disable-next=catching-non-exception
+        except cv2.error as e:
```

Comment:
Why is this specific exception being caught instead of using the broader catch, especially since the code is identical?

Reply:
I did this because of this [comment](https://github.com/UWARG/computer-vision-python/pull/185#discussion_r1667203956). I thought this was needed because `cv2.error` was separate from the `Exception` class but it turns out `cv2.error` inherits from the `Exception` class so I removed it.

Reply:
I don't think it's worth separating them out unless there is a specific use case.